### PR TITLE
Fix some rot in examples/randobj

### PR DESCRIPTION
**Pull request by aboodman**
Also add a buildable file, like examples/demo has.

### CODE REVIEW

File: examples/api/randobj/readme.md [hunk 1]
```diff
@@ -11,14 +11,12 @@ package main
...
-	"os"
 
+	u "github.com/ipfs/go-ipfs-util"
 	"github.com/ipfs/go-ipfs/core"
 	"github.com/ipfs/go-ipfs/core/coreunix"
 	"github.com/ipfs/go-ipfs/repo/fsrepo"
-	u "github.com/ipfs/go-ipfs/util"
-
-	"code.google.com/p/go.net/context"
+	"golang.org/x/net/context"
```
Comment (Kubuxu) [inline examples/api/randobj/readme.md]: use just `"context"` instead

Comment (Kubuxu) [reply examples/api/randobj/readme.md]: If it isn't too big of a refactor.

File: examples/api/randobj/randobj.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+import (
+	"io"
+	"net/http"
+
+	u "github.com/ipfs/go-ipfs-util"
+	"github.com/ipfs/go-ipfs/core"
+	"github.com/ipfs/go-ipfs/core/coreunix"
+	"github.com/ipfs/go-ipfs/repo/fsrepo"
+	"golang.org/x/net/context"
```
Comment (Kubuxu) [inline examples/api/randobj/randobj.go]: yeah, just `context` from stdlib

**Comment by aboodman** [comment]
whoops, ptal

**Comment by aboodman** [comment]
Thanks!

**Pull request merged**